SPECS/rabbitmq-server/rabbitmq-server.spec

```diff
@@ -0,0 +1,106 @@
+%define  debug_package %{nil}
+Summary:        rabbitmq-server
```

Comment:
We would also need an entry in the `cgmanifest.json`

Reply:
License map and `cgmanifest.json` have been updated